gym_xiangqi/xiangqi_game.py

```diff
@@ -200,20 +217,15 @@ def run(self):
         """
         Run the game until terminating condition is achieved
         """
-        if self.on_init(self.agent_piece, self.enemy_piece):
-            clock = pygame.time.Clock()
-            self.running = True
+        clock = pygame.time.Clock()
+        self.running = True
 
         while self.running:
             clock.tick(FPS)
             for event in pygame.event.get():
                 self.on_event(event)
-
             self.render()
 
-        self.game_over()
```

Comment:
Not displaying "gameover" at the end of the game anymore? Then, we can get rid of the method as well maybe?

Reply:
I was figuring out ways to add this into the game loop but the way our code is designed made it tricky. I can remove this right now but if you want, since you are going to add a few more features to GUI, I think it would be nice if you look into existing things like this and try to either use them or remove them.